faculty_mill/faculty_reporter.py

```diff
@@ -77,23 +76,7 @@ def main(
     with tmpdir() as directory:
         directory = Path(directory)
 
-        input_path = directory / "input.ipynb"
-        output_path = directory / "output.ipynb"
-
-        # write the input file to the new file
-        with input_path.open("w") as input_file:
-            shutil.copyfileobj(notebook, input_file)
-
-        if execute:
-            papermill_click_context = papermill.make_context(
-                "The papermill execution command.",
-                [str(input_path), str(output_path)] + click_context.args,
-                parent=click_context,
-            )
-
-            papermill.invoke(papermill_click_context)
-        else:
-            shutil.copy(input_path, output_path)
+        output_path = run(notebook, directory, execute, click_context)
 
         reports = {
             report.name: report for report in report_client.list(PROJECT_ID)
```

Comment:
I don't think PROJECT_ID (and USER_ID) should be capitalized

Reply:
I believe this is addressed in the other PR